template/wpdb-tracy-panel.latte

```diff
@@ -0,0 +1,34 @@
+<h1>DB queries</h1>
+<div class="tracy-inner">
+    <div class="tracy-inner-container">
+        <table class="tracy-sortable">
+            <thead>
+                <tr>
+                    <th>Query</th>
+                    <th>Function list</th>
+                    <th>Time spent (ms)</th>
+                </tr>
+            </thead>
+            <tbody>
+            {foreach $queries as $queryInfo}
+                {if false === $queryInfo}
+                    <tr>
+                        <td colspan="3">Query count exceeded</td>
+                    </tr>
+                    {php break;}
+                {/if}
+
+                <tr>
+                    <td>{$queryInfo['query']}</td>
+                    <td>{$queryInfo['funcList']}</td>
+                    <td>{$queryInfo['timeSpent']|number:10, '.', ''}</td>
```

Comment:
Pokud jsou to milisekundy, zajímalo by mě jen jedno desetinné místo, bylo by to v souladu se zbytkem Tracy.